mobify/progressive-web-scaffold
# ✨ Add SDK components to the project styleguide

Adds the Progressive Web SDK components (as `src/`) to the project styleguide

## Changes
- Add SDK components to project styleguide
- Since SDK components now visible in project styleguide, update project's stylesheet to pull in their base and theme styles
- Clean up README, include styleguide mention

## How to test-drive this PR
- Checkout code
- `npm i`
- `npm run styleguide`
- Visit `http://localhost:4000` to see the new project styleguide
- Add a new project component with `npm run add:component`
- Verify the new project component appears in the project styleguide
